scripts/fetch/fetchGA.R

```diff
@@ -3,21 +3,25 @@ library(googleAnalyticsR)
 library(googleAuthR)
 library(dplyr)
 library(data.table)
-library(sbtools)
 library(assertthat)
+library(aws.s3)
+library(aws.signature)
+
 fetch.GAviews <- function(viz) {
   #first get SB data - does it have data through yesterday? if no, and
   #update is true, run the fetcher for intervening time, and append, upload
 
   if(!viz[['useLocal']]) {
-
+    print(sessionInfo())
     #get file from sb with fetcher
-    viz[['fetcher']] <- 'sciencebase'
-    viz[['remoteFilename']] <- basename(viz[['location']])
-    message("Downloading SB file...")
-    fetch(as.fetcher(viz))
-    message('Downloaded SB file')
-
+    message("Downloading from S3")
+    use_credentials(profile = "default")
```

Comment:
I think we would want the profile to be read in from some environment variable (we could isolate the tiers this way)